tests/baselines/reference/bigintWithLib.errors.txt

```diff
@@ -1,31 +1,11 @@
 bigintWithLib.ts(4,1): error TS2350: Only a void function can be called with the 'new' keyword.
-bigintWithLib.ts(19,33): error TS2769: No overload matches this call.

```

Comment:
I think something is broken here but I'm not sure what yet. A pointer on what is wrong would be appreciated.